tests/suites/test_suite_x509parse.function

```diff
@@ -398,6 +400,32 @@ int parse_crt_ext_cb( void *p_ctx, mbedtls_x509_crt const *crt, mbedtls_x509_buf
  * END_DEPENDENCIES
  */
 
+/* BEGIN_CASE depends_on:MBEDTLS_TEST_HOOKS */
```

Comment:
The CI (more precisely, `tests/scripts/all.sh build_crypto_baremetal`) points out that the dependencies on this test function are not complete. I checked, and it turns out this function also needs to declare a dependency on `MBEDTLS_X509_CRT_PARSE_C`.

(This is a bit surprising, normally we would declare the dependency at the level of the entire test file so that we don't have to repeat it for each function. But it turns out this test file covers things that depend on `MBEDTLS_CRL_PARSE_C` or `MBEDTLS_CSR_PARSE_C`, not necessarily `MBEDTLS_CRT_PARSE_C`, which is why it can't be declared at the file level.)

Reply:
:rocket: